packages/fuels-accounts/src/predicate.rs

```diff
@@ -40,10 +40,17 @@ impl Predicate {
     }
 
     pub fn load_from(file_path: &str) -> Result<Self> {
-        let code = fs::read(file_path).map_err(|e| {
+        let absolute_file_path = path::absolute(file_path).map_err(|e| {
             error!(
                 IO,
-                "could not read predicate binary {file_path:?}. Reason: {e}"
+                "failed to make path absolute: {file_path:?}. Reason: {e}"
+            )
+        })?;
+        let clean_file_path = path_clean::clean(absolute_file_path);
+        let code = fs::read(&clean_file_path).map_err(|e| {
```

Comment:
I'd change this just slightly: 
1. read the code with the original path
2. if that fails, clean up the path and report that we failed to read the clean path
3. if cleaning up fails fallback to the original path and report that

Because we don't need a "clean" path to read the data only for diagnostics. So in the eventuality of any schenanigans with symlinks we'll still function, only our reporting might or might not be nice.

Reply:
`path_clean::clean` returns a `PathBuf`, not a `Result` (can't fail). I agree, though: changed to read with the original path, and report error with cleaned up path in cb4a5cb.